### ISSUE TITLE
Unable to load imports from bower_components 

### ISSUE THREAD

**Issue by htmelvis** [zurb/foundation-cli#18]
I went through the install process using npm and have made sure that everything is up to date. It seems that the setup is unable to resolve imports. It does not appear that ```$.sass({ includePaths: sassPaths}); ``` is functioning when running ```foundation watch``` or ```foundation build```. 

<img width="876" alt="screen shot 2015-11-24 at 1 43 23 pm" src="https://cloud.githubusercontent.com/assets/1952926/11376967/95b8b008-92b1-11e5-91a6-e6938b756f8f.png">

node-sass	3.4.2	(Wrapper)	[JavaScript]
libsass  	3.3.2	(Sass Compiler)	[C/C++]

**Comment by gakimball** [comment]
Did you run `bower install` before starting the build process?

**Comment by htmelvis** [comment]
just tried to run bower install again (I believe it auto ran due to the files being in the dir) but no luck still same issue

**Comment by gakimball** [comment]
See if you have a folder called `bower_components/foundation-sites`. That's where it's trying to pull the Sass from.

**Comment by htmelvis** [comment]
I do and all the assets are in there including the imports which are trying to be loaded

**Comment by kongpda** [comment]
I stuck on this when i run the `foundation watch`.

![untitled-1](https://cloud.githubusercontent.com/assets/8193585/11953644/61545522-a8d4-11e5-9b5d-23da866d9406.png)

**Comment by gakimball** [comment]
Hey friend, really sorry for not getting back to you sooner. We've made a number of improvements to the template Gulpfile, so when you have time, I'd recommend updating your project's `gulpfile.js` and `package.json`, and running `npm install` to update the template. It's been made a lot less error-prone over the last few months.

The "failed to import `util/util`" error means Sass couldn't find the Foundation framework files—that's the first instance where Foundation is imported (in `_settings.scss`), so it's the first chance for an import to fail. There are two main reasons this can happen:

- **No import path is set.** This shouldn't be a problem in our template, as we set up the import path to Foundation for you. Your Sass compiler needs `bower_components/foundation-sites/scss` in its list of import paths. That way, when you write `@import 'util/util'`, it's able to connect the two to load `bower_components/foundation-sites/scss/util/_util.scss`.
- **The files aren't there.** Even if it has the paths it needs, if the files aren't there, then you'll still get an error. In this case, run `bower install` and ensure that you have a `bower_components` folder, and a `foundation-sites` folder inside that.

If you have any more questions, or need any more help, I'm more than happy to lend a hand :) Thanks for using the framework!

I'd also recommend opening an issue on [Foundation for Sites itself](https://github.com/zurb/foundation-sites)—we take issues about the framework files as well as the setup process there.
